www/common.inc

```diff
@@ -260,6 +268,7 @@ if ($privateInstall) {
 } else {
   setcookie('o', $owner, time()+60*60*24*365, '/', 'webpagetest.org');

```

Comment:
My eventual goal for this will be to get ride of `$owner` entirely, only referencing the `User` instance itself. That being said, I'd like to keep cookie setting in the config area of the code. Cookies and anywhere where we mess with the individual user request other than "I grabbed this data" are things I personally like separate.

Longterm, my dream is "use token stored in cookie to grab user object (probably querying an auth service) which has all of this information for us".

Reply:
Makes loads of sense! :)